components/insight/SRC/org/openmicroscopy/shoola/agents/metadata/editor/MapAnnotationsComponent.java

```diff
@@ -665,6 +665,7 @@ private void insertRow() {
 		MapTable t = getSelectedTable();
 		if (t == null)
 			t = getUserTable();
+		if (t == null) return;
```

Comment:
we could check that they are both null i.e. ```getSelectedTable() == null && getUserTable() == null``` return;
but we will still need to assign it if  ```getSelectedTable()``` is ``` null```